starters/adapters/bun/src/entry.bun.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * WHAT IS THIS FILE?
+ *
+ * It's the entry point for the Bun HTTP server when building for production.
+ *
+ * Learn more about the Bun integration here:
+ * - https://qwik.builder.io/docs/deployments/bun/
+ * - https://bun.sh/docs/api/http
+ *
+ */
+import { createQwikCity } from "@builder.io/qwik-city/middleware/bun";
+import qwikCityPlan from "@qwik-city-plan";
+import { manifest } from "@qwik-client-manifest";
+import render from "./entry.ssr";
+
```

Comment:
Is
```ts
declare global {
  interface QwikCityPlatform extends PlatformBun {}
}
```
necessary?

Reply:
It's not in the deno entry file. It may be necessary though, I can add it if needed.

Reply:
It seems to be just a matter of adding
```ts
/**
 * @public
 */
export declare interface PlatformBun {
    ssr?: true;
    // or
    // ssr: boolean;
}
```
in `index.d.ts`.